build-system/tasks/presubmit-checks.js

```diff
@@ -104,6 +104,14 @@ const forbiddenTerms = {
       'build-system/app.js',
       'build-system/amp4test.js',
       'build-system/check-package-manager.js',
+      'build-system/pr-check/build.js',
+      'build-system/pr-check/checks.js',
+      'build-system/pr-check/dist-test.js',
+      'build-system/pr-check/local-test.js',
+      'build-system/pr-check/remote-test.js',
+      'build-system/pr-check/utils.js',
+      'build-system/pr-check/validator.js',
+      'build-system/pr-check/visual-diff-test.js',
```

Comment:
Grammar nit: For consistency, pluralize `tests` in all these filenames, just like `checks` and `utils` are pluralized.
```
      'build-system/pr-check/dist-tests.js',
      'build-system/pr-check/local-tests.js',
      'build-system/pr-check/remote-tests.js',
      'build-system/pr-check/validator-tests.js',
      'build-system/pr-check/visual-diff-tests.js',
```